### PR TITLE
update event

### PR DESCRIPTION
**Pull request by Touriist**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 4d6c73108858642cc6925e8e1859be416e0db591 at: https://66f7a81a92c9be839c088a8e--layer5.netlify.app

File: src/collections/events/2024/hacktoberfest-2024/index.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ register: true
 ---
 
 import { Link } from "gatsby" ;
-
+import hacktoberfestprepslides from "./hacktoberfest-prep-2024-extending-meshery-models.pdf"
```
Comment (Touriist) [inline src/collections/events/2024/hacktoberfest-2024/index.mdx:16]:
```suggestion
import preptemberslides from "./hacktoberfest-prep-2024-extending-meshery-models.pdf"
```

File: src/collections/events/2024/hacktoberfest-2024/index.mdx [hunk 2]
```diff
@@ -40,5 +40,6 @@ Prepare for Hacktoberfest and ready yourself for contributing to CNCF projects t
 <ul>
     <li><Link to="/blog/open-source/ways-to-contribute-at-layer5">Code and Non-Code Contributing at Layer5</Link></li>
     <li><a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+org%3Ameshery+org%3Aservice-mesh-performance+org%3Aservice-mesh-patterns+label%3A%22hacktoberfest%22+">Issues for Hacktoberfest</a></li>
-    {/* <li><a href="https://www.youtube.com/watch?v=T7xLUqrnxtI">Recorded Livestream</a></li> */}
+    <li><a href="https://www.youtube.com/watch?v=iKe52yMWWK4">Recorded Livestream</a></li>
+    <li><a href={hacktoberfestprepslides}>Presentation Slides</a></li>
```
Comment (Touriist) [inline src/collections/events/2024/hacktoberfest-2024/index.mdx:44]:
```suggestion
    <li><a href={preptemberslides}>Presentation Slides</a></li>
```

**Comment by l5io** [comment]
🚀 Preview for commit 80a4e6387a25fe20d44872279fd0d983a3daaed4 at: https://66f7adb23c30e07266339284--layer5.netlify.app

**Pull request merged**